### PR TITLE
Added a new mentor

### LINKED ISSUE #96: Some mentors are missing from the Mentors section
**Actual Behaviour**
* Some mentors are missing from the Mentors section

**Expected Behaviour**
* All mentors should be present in the PR

List of mentors - 
- [x] Shubham Kumar
- [x] Harshit Khandelwal
- [x] tabesin
- [x] Shashank Kumar
- [x] Saurabh Srivastava 
- [x] Akshat Garg
- [x] Arundhati Gupta
- [x] Jogendra Singh
- [x] Anshuman Verma
- [x] Sanskar Jethi
- [x] Akshat Jain
- [x] Mohit Sharma
- [x] Prateek Chanda
- [x] Mohit Kumar
- [ ] Hon Nguyen
- [x] Abishek V Ashok
- [x] Rohit Potter
- [x] Shubham Gupta
- [x] Nikit Bhandari
- [ ] Tarun Kumar
- [x] Sridhar Jajoo 
- [ ] Simranpreet kaur

Note - Please add 1 mentor per PR so that all the students get a chance to work on it.

### PR DESCRIPTION
**Pull request by midhmanohar**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://midhmanohar.github.io/gci18.fossasia.org/)

- Fixes #96 

## Description / Changes

added mentor Prateek Chanda.

## Screenshots if any:
pc

- - - - - - - - - - - -

### CODE REVIEW

**Comment by midhmanohar** [comment]
i will update it

**Comment by midhmanohar** [comment]
i will update it

**Comment by midhmanohar** [comment]
i have updated it.

File: _data/mentors.yml [hunk 1]
```diff
@@ -108,3 +108,7 @@
 - name: Tarun Kumar
   image: Tarun Kumar.jpg
   github: meets2tarun
+
+- name: Prateek Chanda
+  image: prateekiiest.jpg
+  github: prateekiiest
```
Comment (Ryan10145) [inline _data/mentors.yml]: Can you add a new line at the end of the file?

**Pull request merged**